routers/likes.py

```
@@ -0,0 +1,165 @@
+        # 취소할 좋아요가 없어도 204로 멱등 처리
+        write_json("likes.json", likes)  # 없어도 됨
+        return
+    write_json("likes.json", likes)
+    return
+
+# 좋아요 상태 확인
+@router.get("/users/me/posts/likes/summary", response_model=LikeOut)
+def my_likes_summary(
+    from_: Optional[str] = None,
+    to: Optional[str] = None,
+    user=Depends(get_current_user)
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)